File: Lab 2/C#/main.cs
using System;

internal class Program
{
    static void Main()
    {
        
        Console.Write("Enter x1: "); double x1 = Convert.ToDouble(Console.ReadLine());
        Console.Write("Enter y1: "); double y1 = Convert.ToDouble(Console.ReadLine());
        Console.Write("Enter x2: "); double x2 = Convert.ToDouble(Console.ReadLine());
        Console.Write("Enter y2: "); double y2 = Convert.ToDouble(Console.ReadLine());

        LineSegment segment = new LineSegment(x1, y1, x2, y2);

        Console.WriteLine("--------------------------------");
        Console.WriteLine($"X1: {x1} Y1: {y1}");
        Console.WriteLine($"X2: {x2} Y2: {y2}");
        Console.WriteLine("--------------------------------");
        Console.WriteLine($"Segment length is: {segment.Length():F2}");
        Console.WriteLine($"Angle with OX is: {segment.AngleOX():F2} degrees");
        Console.WriteLine("--------------------------------");

    }
}

File: Lab 2/C++/main.cpp
#include <iostream>
#include <iomanip>
#include "ClassLib.h"
using namespace std;

int main() 
{
    double x1, y1, x2, y2;

    cout << "Enter x1: ";
    cin >> x1;
    cout << "Enter y1: ";
    cin >> y1;
    cout << "Enter x2: ";
    cin >> x2;
    cout << "Enter y2: ";
    cin >> y2;

    LineSegment segment(x1, y1, x2, y2);

    cout << "--------------------------------" << endl;
    cout << "X1: " << x1 << " Y1: " << y1 << endl;
    cout << "X2: " << x2 << " Y2: " << y2 << endl;
    cout << "--------------------------------" << endl;
    cout << "Segment length is: " << segment.length() << endl;
    cout << "Angle with OX is: " << segment.angleOX() << endl;
    cout << "--------------------------------" << endl;

    return 0;
}
